### PR TITLE
Add team leads to about page (#140).

### LINKED ISSUE #140: Add team leads to about page
Let's add our volunteer team leaders to the About page!

### PR DESCRIPTION
**Pull request by renderf0x**
## Resolves #140

This is a basic version of a card-based team-leads listing for the site.
Right now, only Ann has a custom pic; for everyone else it's defaulting back to your github profile.
Also, I don't know Teresa's last name...can someone provide this?

In the future, I think we should add another card that displays a mini-grid of recent contributors to include more of the community, but that can wait for a future PR.

### How to test

1. Stand up dev stack and go to about page
2. Confirm that cards appear for devs near the bottom
3. Switch to mobile emulation and test responsiveness

### Screenshots

スクリーンショット 2021-06-27 17 28 02
スクリーンショット 2021-06-27 17 29 22

### CODE REVIEW

**Comment by renderf0x** [comment]
For this PR, if you want custom info for the following, please let me know...

Right now only @ann-kilzer has a custom image.
If you want one @theyokohamalife @Ermish @thehamspecial let me know.

Also, I only know twitter handles for LaShawn and Ann, so @Ermish and @thehamspecial if you have them let me know.

And lastly, @thehamspecial  - I don't know your last name for this. Do you want it on there or do you just want to be 'Teresa'?

**Comment by Ermish** [comment]
Here's my pic!
Image from iOS

File: components/ContributorLead.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+</template>
+<script>
+
+const getGithubProfileUrl = function(handle){
+    return `https://github.com/${handle}`;
+}
+
+export default {
+    props: {
+        githubHandle: String,
```
Comment (ann-kilzer) [inline components/ContributorLead.vue:55]: It's a good vue practice to either give a default value or make props required

File: data/contributors.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          "name": "LaShawn Toyoda",
+          "githubHandle": "theyokohamalife",
+          "twitterHandle": "theyokohamalife",
+          "role": "Project Originator and Lead Developer"
+        },
+        {
+          "name": "Ann Kilzer",
+          "githubHandle": "ann-kilzer",
+          "twitterHandle": "stonecoldkilzer",
+          "role": "Dev, QA, and localization design",
```
Comment (ann-kilzer) [inline data/contributors.json:13]:
```suggestion
          "role": "Dev, tooling, and localization design",
```

This is a little closer to my role, as if I were *really doing QA* I'd insist on adding a testing framework 😆 

No time 😆

File: pages/about.vue [hunk 3]
```diff
@@ -24,6 +24,10 @@
         <h2>{{ $t("about.supportQ") }}</h2>
         {{ $t("about.supportA") }}
       </div>
+      <div>
+        <h2>Who developed this site?</h2>
```
Comment (ann-kilzer) [inline pages/about.vue:28]: If there's time, can we use i18n? (But it would be best to get this in by tomorrow morning so it can come later)

**Review by ann-kilzer — APPROVED** [review]
LGTM 🖼️ 

with some suggestions

File: pages/about.vue (hunk 3 above)
Comment (renderf0x) [reply pages/about.vue:28]: Yeah, exactly why I didn't do that yet. I'll merge this first, then make immediate PRs to get the key in.

File: data/contributors.json (hunk 2 above)
Comment (renderf0x) [reply data/contributors.json:13]: sure =). Any adjustments for anyone else?

**Comment by renderf0x** [comment]
> Here's my pic!


Rockstar, thanks @Ermish

File: components/ContributorLead.vue (hunk 1 above)
Comment (renderf0x) [reply components/ContributorLead.vue:55]: Cool, default values don't really make sense here, but I'll make the ones that are required...required =).

File: data/contributors.json (hunk 2 above)
Comment (renderf0x) [reply data/contributors.json:13]: haha. I'd say that's SDET rather than QA but now we're splitting hairs =). I'll change it

**Pull request merged**